README.md

```diff
@@ -1,2 +1,55 @@
-# anbox-cloud-demos
-WIP
\ No newline at end of file
+# Anbox Cloud Demos
+
+A repo to showcase how android apps can use Anbox Cloud Appliance for local development, and to demo how Anbox can be part of github workflows for running ui screenshots tests and generating test reports.
+
+It includes [./nowinandroid](https://github.com/android/nowinandroid) as an example of a typical android project.
+
+## Installation
+
+- Install [Android Studio](https://developer.android.com/studio) or VScode with all relevant dependencies and extensions, also make sure to add the following to your `~/.bashrc`
+
+```
+export PATH="~$PATH:~/Android/Sdk/platform-tools"
+export ANDROID_HOME=$HOME/Android/Sdk
+export PATH=$ANDROID_HOME/cmdline-tools/latest/bin:$PATH
+export PATH=$ANDROID_HOME/emulator:$PATH
+export PATH=$ANDROID_HOME/platform-tools:$PATH
+```
+
+## Development
+
+- Clone the repo and open the root directory in Android Studio or VScode.
+- Run `make nia-build | nia-install` to buil and generate nia app apk or install the apk - as 'DemoDebug' flavor - directly on the connected device.
```

Comment:
I think we should explicitly tell users that `nia` stands for `nowinandroid`, to be sure it's super clear.